README.md

```diff
@@ -381,135 +309,88 @@ try {
 } catch {}
 ```
 
-Complete discussion about this topic at https://github.com/arthurfiorette/proposal-safe-assignment-operator/issues/13 if the reader is interested.
-
-<br />
-
-## Polyfilling
-
-This proposal can be polyfilled using the code provided at [`polyfill.js`](./polyfill.js).
-
-However, the `?=` operator itself cannot be polyfilled directly. When targeting older JavaScript environments, a post-processor should be used to transform the `?=` operator into the corresponding `[Symbol.result]` calls.
-
-```ts
-const [error, data] ?= await asyncAction(arg1, arg2)
-// should become
-const [error, data] = await asyncAction[Symbol.result](arg1, arg2)
-```
-
-```ts
-const [error, data] ?= action()
-// should become
-const [error, data] = action[Symbol.result]()
-```
-
-```ts
-const [error, data] ?= obj
-// should become
-const [error, data] = obj[Symbol.result]()
-```
+A detailed discussion about this topic is available at [GitHub Issue #13](https://github.com/arthurfiorette/try-expressions/issues/13) for those interested.
```

Comment:
```suggestion
A detailed discussion about this topic is available at [GitHub Issue #13](https://github.com/arthurfiorette/proposal-safe-assignment-operator/issues/13) for those interested.
```

Reply:
I'll be renaming this repo when this gets merged